src/index.js

```diff
@@ -5,30 +5,56 @@ import '@fortawesome/fontawesome-free/js/regular';
 import '@fortawesome/fontawesome-free/js/brands';
 import Storage from './storage.js';
 import Score from './score.js';
-import Display from './display';
+import Display from './display.js';
 
 const lbName = document.querySelector('.lb-name');
 const lbScore = document.querySelector('.lb-score');
 
 const scoreAdd = document.querySelector('.score-add');
 const scoreRefresh = document.querySelector('.score-refresh');
 
-document.addEventListener(
-  'DOMContentLoaded',
-  Display.displayList(Storage.getList().list),
-);
+document.addEventListener('DOMContentLoaded', Display.displayList());
+
```

Comment:
- [ ] I would recommend aligning the result data and follow the wireframe design so that it looks nice. Please fix it.

### Your design 

![Screenshot from 2021-09-04 01-43-47](https://user-images.githubusercontent.com/22434660/132061303-7af56694-6437-40b6-99c7-80cea2e60ce1.png)


### Wireframe design

![Screenshot from 2021-09-01 05-05-45](https://user-images.githubusercontent.com/22434660/132061299-f664cf18-4094-46fb-bf20-3054b3f48bd5.png)